
# HG changeset patch
# User Mike Hommey <mh+mozilla@glandium.org>
# Date 1728937349 0
# Node ID 62e0dd5ec3d511e55137da634460d31b9214e24a
# Parent  d369655cf80acb2b00ba5c0dbf78792714b60a51
Bug 1924031 - Replace use of pipes.quote with shlex.quote. r=taskgraph-reviewers,jcristau

It was literally a reexport before the pipes module was removed in
python 3.13.
https://github.com/python/cpython/blob/3.12/Lib/pipes.py#L66

Differential Revision: https://phabricator.services.mozilla.com/D225319

diff --git a/taskcluster/android_taskgraph/job.py b/taskcluster/android_taskgraph/job.py
--- a/taskcluster/android_taskgraph/job.py
+++ b/taskcluster/android_taskgraph/job.py
@@ -1,14 +1,14 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 
-from pipes import quote as shell_quote
+from shlex import quote as shell_quote
 
 from gecko_taskgraph.transforms.job import configure_taskdesc_for_run, run_job_using
 from taskgraph.util import path
 from taskgraph.util.schema import Schema, taskref_or_string
 from voluptuous import Optional, Required
 
 secret_schema = {
     Required("name"): str,

